### DISCUSSION TITLE
[Feature]: simplified Analytics Integration with Connectors

### DISCUSSION THREAD

**Discussion by moskalakamil** [TheWidlarzGroup/react-native-video#4991] [Ideas]
### What problem are you trying to solve?

Currently, you can implement analytics using the [Plugin](https://docs.thewidlarzgroup.com/react-native-video/other/plugin). This approach requires setting up analytics natively by connecting separately to ExoPlayer and AVPlayer, which involves knowledge of Kotlin and Swift.

### Proposed solution

A dedicated connector that simplifies analytics integration, enabling analytics implementation with just a few lines of JavaScript or TypeScript. Contact us for support with tools like Conviva, Adobe, Comscore, Agama, or others.

### Platforms

iOS, Android

#### Support

If this functionality is important to you and you need it, contact [TheWidlarzGroup](https://thewidlarzgroup.com/) - [hi@thewidlarzgroup.com](mailto:hi@thewidlarzgroup.com)

[a comment by an automated account is not shown]

**Comment by TWG-AI** [comment]
Thanks for the detailed proposal! A plugin-based analytics connector that abstracts the native ExoPlayer/AVPlayer analytics hooks would meaningfully lower the bar for analytics integration. We've noted this for the team — it aligns well with the v7 plugin architecture direction.
